Modules/Retailer/Routes/api.php

```diff
@@ -13,16 +13,16 @@
 | is assigned the "api" middleware group. Enjoy building your API!
 |
 */
-Route::prefix('retailer')->group(function () {
+Route::prefix('retailers')->group(function () {
```

Comment:
This group and next group should be merged, keep only one group which will contain reset_password and store.